src/kubernetes/argo/patch.diff

```diff
@@ -0,0 +1,3302 @@
+diff --git a/kubernetes/argo/defaults.dhall b/kubernetes/argo/defaults.dhall
```

Comment:
is there any point in having this patch file here?

Reply:
It was to keep the changes we did for argo workflow visible and make it usable with `git patch`

Reply:
Right but does it make sense to have it in both places?

Reply:
For sure it makes sense in `src/`. I'll look into removing it from `/kubernetes`